Python/Product/Analysis/LanguageServer/Server.WorkspaceSymbols.cs

```diff
@@ -78,7 +78,7 @@ string prefix
             var loc = m.Locations.FirstOrDefault(l => !string.IsNullOrEmpty(l.FilePath));
             if (loc != null) {
                 res.location = new Location {
-                    uri = new Uri(PathUtils.NormalizePath(loc.FilePath), UriKind.RelativeOrAbsolute),
+                    uri = new Uri(PathUtils.NormalizePath(loc.FilePath), UriKind.Absolute),
```

Comment:
Since we're converting from a file path here, we can check whether it's rooted before crashing when it's not.

Reply:
Or better yet, just use the `DocumentUri` member of the `LocationInfo`.

Reply:
Good point.